### PR TITLE
fix icon base_camp

### PR DESCRIPTION
**Pull request by ginold**

### CODE REVIEW

File: c2corg_ui/static/img/documents/waypoints/base_camp.svg [hunk 1]
```diff
@@ -12,26 +12,18 @@
    width="65"
    height="65"
    viewBox="0 0 65.000001 65.000001"
-   id="svg4604"
+   id="svg2"
    version="1.1"
    inkscape:version="0.91 r13725"
-   sodipodi:docname="camping.svg">
+   sodipodi:docname="camp_site.svg">
```
Comment (asaunier) [inline c2corg_ui/static/img/documents/waypoints/base_camp.svg]: Do you mean "base_camp"?

File: c2corg_ui/static/img/documents/waypoints/camp_site.svg [hunk 2]
```diff
@@ -12,18 +12,26 @@
    width="65"
    height="65"
    viewBox="0 0 65.000001 65.000001"
-   id="svg2"
+   id="svg4604"
    version="1.1"
    inkscape:version="0.91 r13725"
-   sodipodi:docname="camp_site.svg">
+   sodipodi:docname="camping.svg">
```
Comment (asaunier) [inline c2corg_ui/static/img/documents/waypoints/camp_site.svg]: Shouldn't it be "camp_site" as the filename?

Comment (ginold) [inline c2corg_ui/static/img/documents/waypoints/camp_site.svg]: i didn't notice, it replaced itself !

**Comment by ginold** [comment]
fixed. ok for merging?

**Comment by asaunier** [comment]
Please do

**Pull request merged**